### PR TITLE
added data and documentation for media mentions (cable and online) data

### PR DESCRIPTION
**Pull request by niannucci**
I added the media_mentions_cable and media_mentions_online datasets and accompanying documentation to the FiveThirtyEight package.

### CODE REVIEW

File: R/data_natalia.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#' \describe{
+#'   \item{date}{start date for the week of coverage}
+#'   \item{name}{candidate's name}
+#'   \item{matched_stories}{number of stories in that week that mention the specified candidate}
+#'   \item{all_candidate_stories}{number of stories in that week that mention any candidate}
+#'   \item{pct_of_all_candidate_stories}{percentage of stories in which that specific candidate is mentioned out of all stories mentioning any candidate for that week (matched_stories / all_candidate_stories)}
+#'   \item{query}{query for Media Cloud}
+#' }
+#'@source Two collections in the Media Cloud \url{https://mediacloud.org/} database U.S. Top Online News \url{https://sources.mediacloud.org/#/collections/58722749} and U.S. Top Digital Native News \url{https://sources.mediacloud.org/#/collections/57078150}
+"media_mentions_online"
```
Comment (beanumber) [inline R/data_natalia.R]: Can you please add a new line at the end? [Just hit return.]

File: R/data_natalia.R [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#'   \item{matched_clips}{number of 15-second clips in that week that mention the specified candidate}
+#'   \item{all_candidate_clips}{number of 15-second clips in that week that mention any candidates}
+#'   \item{total_clips}{total number of 15-second clips that week across the three networks}
+#'   \item{pct_of_all_candidate_clips}{percentage of clips in which that specific candidate is mentioned out of all clips mentioning any candidate for that week (matched_clips / all_candidate_clips)}
+#'   \item{query}{query used for the GDELT Television API}
+#' }
+#' @source The GDELT Television API \url{https://blog.gdeltproject.org/gdelt-2-0-television-api-debuts/}, which processes the data from the TV News Archive \url{https://archive.org/details/tv}.
+"media_mentions_cable"
+
+
```
Comment (beanumber) [inline R/data_natalia.R]: @rudeboybert do you want these to appear together in the documentation? Perhaps `@rdname` would be appropriate here?

**Review by beanumber — CHANGES_REQUESTED** [review]

File: R/data_natalia.R (hunk 2 above)
Comment (niannucci) [reply R/data_natalia.R]: @beanumber is there a way I could do this, as each of the two datasets has different variables? I tried doing this, but I can only get the format section to show up in the documentation for the first dataset, as they each need different format/describe documentation.

Comment (beanumber) [reply R/data_natalia.R]: You should be able to include the documentation for both data sets in the same `man` file. See (http://r-pkgs.had.co.nz/man.html#multiple-man).

**Pull request closed without merging**